databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -236,6 +246,16 @@ def get_recipient_cli(api_client, name):
                short_help='Update a recipient.')
 @click.option('--name', required=True,
               help='Name of the recipient who needs to be updated.')
+@click.option('--new-name', default=None, help='New name of the recipient.')
+@click.option('--comment', default=None, required=False,
+              help='Free-form text description.')
+@click.option('--owner', default=None, required=False,
+              help='Owner of the recipient.')
+@click.option('--allowed-ip-address', default=None, required=False, multiple=True,
+              help=(
+                  'IP address in CIDR notation that is allowed to use delta sharing '
+                  '(can be specified multiple times). Specify a single empty string to disable '
+                  'IP allowlist.'))
```

Comment:
This is a bad API, IMO. If the intent is to clear the allowed IP addresses field to delete them, there should be a dedicated option, e.g. `--clear-allowed-ip-addresses` to do so. Using a magic empty string can lead to incorrect usage where the list is cleared accidentally (e.g. bad interpolation in a script).